mne/io/nihon/nihon.py

```diff
@@ -409,7 +426,10 @@ def __init__(self, fname, preload=False, verbose=None):
                 annots['description'].append('EDGE boundary')
 
         annotations = Annotations(**annots, orig_time=info['meas_date'])
-        self.set_annotations(annotations)
+        if header['version'] == _valid_headers[-1]:
+            self._annotations = annotations
+        else:
+            self.set_annotations(annotations)
```

Comment:
I'm still a bit wary of this change and would like input from other devs. `set_annotations` has some sensible sanity/safety checks and I don't like the idea of skipping them; I'd rather figure out why things aren't working and fix them.

Reply:
After looking into this more closely. I believe this has to do with the multiple blocks of data. There is only one collected (due to the version), so the rest of the annotations would be seen as out of range for the data found. Skipping this would allow to get the annotations even though we have no data for those annotations.

I am not sure that is helpful to have the annotations without the observations. I will spend sometime trying to get all of those.